components/SearchList.js

```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
+
+export default function SearchList({ result, handleClick }) {
+  return (
+    <>
+      <StyledSearchListWrapper>
+        <StyledSearchList>
```

Comment:
```suggestion
        <StyledSearchList role="list">
```

Reply:
Thank you for your comment, Sven! But I guess that the role attribute has no impact in this context. "StyledSearchList" is a react component (a styled one, but a react component). Means that it's not an element of the DOM but a virtual DOM element (so this attribute is not really attached). And the StyledSearchList is a semantic ul/li. So I guess I don't have to specify the role.

Reply:
My pleasure :)
If I remember correctly the role is important then when list style is set to none that screenreader recognizes the styledlist as a list as well. Please correct me if I am wrong :)